src/test/kotlin/org/jetbrains/research/ml/ast/transformations/util/TransformationsTestHelper.kt

```diff
@@ -61,14 +57,28 @@ object TransformationsTestHelper {
         val expectedSrc = expectedPsiInFile.text
         logger.info("The expected code is:\n$expectedSrc")
         ApplicationManager.getApplication().invokeAndWait {
-            transformation(psiInFile, true)
-            if (toCheckFileStructure) {
-                PsiTestUtil.checkFileStructure(psiInFile)
-            }
+            transformation(psiInFile, cs)
+            PsiTestUtil.checkFileStructure(psiInFile)
         }
         fileHandler.formatPsiFile(psiInFile)
         val actualSrc = psiInFile.text
         logger.info("The actual code is:\n$actualSrc")
         BasePlatformTestCase.assertEquals(expectedSrc, actualSrc)
     }
+
+    fun getForwardTransformationWrapper(
+        forwardTransformation: (PsiElement, cs: PerformedCommandStorage?) -> Unit,
+        commandStorage: PerformedCommandStorage? = null
+    ): TransformationDelayed =
+        { psi: PsiElement, _: PerformedCommandStorage? -> forwardTransformation(psi, commandStorage) }
+
+    fun getBackwardTransformationWrapper(
+        forwardTransformation: (PsiElement, cs: PerformedCommandStorage?) -> Unit
+    ): TransformationDelayed =
+        { psi: PsiElement, _: PerformedCommandStorage? ->
+            val commandStorage = PerformedCommandStorage(psi)
+            getForwardTransformationWrapper(forwardTransformation, commandStorage)(psi, commandStorage)
```

Comment:
Why do we need to use a wrapper here? Cannot we just call forwardTransformation?